src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$main-text-color: #060b35;
+$main-text-color-hover: #34568b;
+$secundary-text-color: #616070;
```

Comment:
There is a typo in the variable name `$secundary-text-color`. It should be `$secondary-text-color` to follow standard spelling conventions.